russstation/code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -27,4 +27,16 @@
 		M.fakevomit(0)
 	return ..()
 
+
+/datum/reagent/consumable/superlaughter/traitor_pen_variant
```

Comment:
"variant" strikes me as a bit redundant, considering as a subtype it's by nature a variant of it's supertype